src/TimeoutMigrationTool.SqlP.AcceptanceTests/SqlPAcceptanceTest.cs

```diff
@@ -30,7 +30,7 @@ public async Task SetUp()
 
                 testName = testName.Replace("_", "");
 
-                return testName + "_" + endpointBuilder;
+                return testName + "-" + endpointBuilder;
```

Comment:
Ok so this is that we get valid endpoint name prefixes for ASQ?

Reply:
Yes, `_` doesn't seem to be liked